### PR TITLE
Breaks out the Talks sections

### PR DESCRIPTION
**Pull request by ludgerpaehler**
Breaking out the talks and related publications header into its individual sections

### CODE REVIEW

**Comment by ludgerpaehler** [comment]
Note to self: headers to be updated to point to EnzymeAD, and `Conference` header to be added. `Source` -> `GitHub`

File: content/talks/EnzymeCon.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Enzyme Conference 2023"
+date: 2019-11-29T15:26:15Z
```
Comment (tgymnich) [inline content/talks/EnzymeCon.md]: whats up with the date here?

File: content/talks/Publications.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Related Publications"
+date: 2019-11-29T15:26:15Z
```
Comment (tgymnich) [inline content/talks/Publications.md]: whats up with the date here?

File: content/talks/Talks.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+title: "Presentations"
+date: 2019-11-29T15:26:15Z
```
Comment (tgymnich) [inline content/talks/Talks.md]: date

**Review by tgymnich — APPROVED** [review]

File: content/talks/EnzymeCon.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+draft: false
+weight: 25
+---
+
+## About
+
+The first [Enzyme workshop](https://enzyme.mit.edu/conference) took place from February 22-24 in Boulder, Co. The workshop's aim was to bring together the contributors and users of Enzyme in particular, and people interested in automatic differentiation within the LLVM infrastructure in general, for two days of fast-paced high-bandwidth idea exchange and collaboration bootstrapping. For this the workshop was designed to have a flexible format, friendly to practitioners and researchers alike; its program consisted of a mix of presentations, tutorials, roundtable discussions and live programming sessions.
```
Comment (wsmoses) [inline content/talks/EnzymeCon.md:14]: Can we add the conference photo here?

File: content/talks/Talks.md (hunk 3 above)
Comment (ludgerpaehler) [reply content/talks/Talks.md]: Changed.

File: content/talks/EnzymeCon.md (hunk 1 above)
Comment (ludgerpaehler) [reply content/talks/EnzymeCon.md]: Changed.

File: content/talks/EnzymeCon.md (hunk 4 above)
Comment (ludgerpaehler) [reply content/talks/EnzymeCon.md:14]: Absolutely! Done.

File: content/talks/Publications.md (hunk 2 above)
Comment (ludgerpaehler) [reply content/talks/Publications.md]: Changed.

**Pull request merged**